README_E1480

```diff
@@ -0,0 +1,17 @@
+E1480 - Test the student interface
```

Comment:
The README has been removed.

On Fri, Jan 23, 2015 at 1:35 PM, Andrew Kofink <notifications@github.com>
wrote:

> In README_E1480
> <https://github.com/expertiza/expertiza/pull/468#discussion_r23469555>:
>
> > @@ -0,0 +1,17 @@
> > +E1480 - Test the student interface
>
> Remove this readme
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/expertiza/expertiza/pull/468/files#r23469555>.
>